mk/spksrc.cross-rust.mk

```diff
@@ -53,10 +53,14 @@ endif
 CARGO_INSTALL_ARGS += --path $(RUST_SRC_DIR)
 CARGO_INSTALL_ARGS += --root $(STAGING_INSTALL_PREFIX)
 
+ifeq ($(strip $(CARGO_BUILD_ARGS)),)
+CARGO_BUILD_ARGS = 
+endif
+
 # Default build with rust and install with cargo
 rust_install_target:
 	@echo "  ==> Cargo install rust package $(PKG_NAME) ($(shell rustc --version); $(RUST_TOOLCHAIN))"
-	@$(RUN) cargo +$(RUST_TOOLCHAIN) install $(CARGO_INSTALL_ARGS)
+	@$(RUN) cargo +$(RUST_TOOLCHAIN) install $(CARGO_INSTALL_ARGS) $(CARGO_BUILD_ARGS)
```

Comment:
Approch is good.  Is the CARGO_BUILD_ARGS a default supported cargo environment variable?  Or if only used within spksrc then ensure it's naming is aligned with other similar case (such as CMAKE for instance).

Another approaches could have been to simply call `CARGO_INSTALL_ARGS += <myvar>`.  If `CARGO_BUILD_ARGS` is a standard cargo variable then you could as well use a `ENV += CARGO_BUILD_ARGS=<myvar>`.

Reply:
Based on the cargo [documentation](https://doc.rust-lang.org/cargo/reference/environment-variables.html), `CARGO_BUILD_ARGS` is not an environment variable for cargo but rather something that is specific to spksrc. I saw it used in someone else's code in the repo so I thought it would be a good way to enable that.

You are correct that a simpler way to append this would be:
```
ifeq ($(strip $(CARGO_BUILD_ARGS)),)
CARGO_BUILD_ARGS = 
else
CARGO_INSTALL_ARGS += CARGO_BUILD_ARGS
endif

Then there would be no reason to modify the execution (RUN) line.